src/drc/api/filters.py

```diff
@@ -13,9 +12,15 @@
 
 
 class EnkelvoudigInformatieObjectListFilter(FilterSet):
+    def zoek_uuids(self, queryset, name, values):
```

Comment:
Ik denk dat je dit stukje niet nodig hebt als je gebruikt maakt van bijvoorbeeld de `UUIDFilter` in combinatie met `field_name` keyword argument. Dat zou hetzelfde moeten doen als wat je hier doet zonder dat wij zelf extra logica toe hoeven te voegen.

Reply:
Dit had ik geprobeerd alleen lukte het mij niet zo te filteren op basis van een array input. 

uuid = filters.UUIDFilter(field_name="uuid", lookup_expr="contains")

Met deze krijg ik een len(response_data) van 1. 

Of doe ik dan wat verkeerds?

Reply:
De `lookup_expr` op `contains` zou ik achterwegen laten (of veranderen naar `exact` volgens mij?), de huidige code heeft dat gedrag ook niet.